app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -22,15 +22,15 @@ class MainActivity : ComponentActivity() {
                 // A surface container using the 'background' color from the theme
                 Surface(color = MaterialTheme.colors.background,
                 modifier = Modifier.fillMaxWidth()) {
-                    RecycleSearch("Android")
+                    recycleSearch("Android")
```

Comment:
Function names in Kotlin should use camel case, which means starting the function name with a lowercase letter. I believe the document states that only unit test functions should start with a capital letter

Reply:
Normally I'd agree with this, but... there's an exception for Compose.  Compose functions should be nouns, and should be PascalCase. 
https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md